client/state/themes/actions/theme-transfer.js

```diff
@@ -153,11 +153,18 @@ function transferInitiateFailure( siteId, error, plugin, context ) {
  * to avoid unhandled rejections in production.
  * @param {number} siteId -- the site being transferred
  * @param {number} transferId -- the specific transfer
+ * @param {boolean} hasThemeFileUpload -- has theme File Upload
  * @param {number} [interval] -- time between poll attempts
  * @param {number} [timeout] -- time to wait for 'complete' status before bailing
  * @returns {Promise} for testing purposes only
  */
-export function pollThemeTransferStatus( siteId, transferId, interval = 3000, timeout = 180000 ) {
+export function pollThemeTransferStatus(
+	siteId,
+	transferId,
+	hasThemeFileUpload = false,
```

Comment:
Can this new param be the last param? Unless we're 100% sure that every single place calling this function is using the new signature.

Reply:
Done :)